tests/textlab_api/textlab_api_test.py

```diff
@@ -0,0 +1,302 @@
+
+    # Check the updated HIX score
+    page_translation = Page.objects.get(id=2).get_translation("de")
+
+    assert page_translation.hix_score == 15.48920568
+    assert mock_server.requests_counter == 2
+
+
+@pytest.mark.django_db
+def test_hix_disable_on_region(
+    load_test_data: None,
+    admin_client: Client,
+    settings: SettingsWrapper,
+    mock_server: MockServer,
+) -> None:
+    """
+    Check that the HIX score is not updated when hix is disabled on region level. No request to TextLab API is expected.
+
+    :param load_test_data: The fixture providing the test data (see :meth:`~tests.conftest.load_test_data`)
+    :param admin_client: The fixture providing the http client
+    :param settings: The fixture providing the django settings
+    :param mock_server: The fixture providing the dummy http server
+    """
+
+    # Setup a mocked Textlab API server with dummy responses
+    mock_server.configure("/user/login", 200, {"token": "dummy"})
+    mock_server.configure("/benchmark/420", 200, {"formulaHix": 20.0})
+
+    # Redirect call aimed at the Textlab API to the fake server
+    settings.TEXTLAB_API_URL = f"http://localhost:{mock_server.port}"
+
+    # Disable Textlab in the test region
+    Region.objects.filter(slug="augsburg").update(hix_enabled=False)
+
+    edit_page, response = update_page_content(
+        admin_client, "Willkommen in Augsburg", "Neuer Inhalt1"
+    )
+
+    assert response.status_code == 302
+    assert response.headers.get("Location") == edit_page
+
+    # Check that HIX score was not updated
+    page_translation = Page.objects.get(id=2).get_translation("de")
+
+    assert page_translation.hix_score is None
+    assert mock_server.requests_counter == 0
+
+
+@pytest.mark.django_db
+def test_ignore_hix_on_page(
+    load_test_data: None,
+    admin_client: Client,
+    settings: SettingsWrapper,
+    mock_server: MockServer,
+) -> None:
+    """
+    Check that the HIX score is not updated when hix is disabled at the page level (hix_ignore=True). No request to TextLab API is expected.
+
+    :param load_test_data: The fixture providing the test data (see :meth:`~tests.conftest.load_test_data`)
+    :param admin_client: The fixture providing the http client
+    :param settings: The fixture providing the django settings
+    :param mock_server: The fixture providing the dummy http server
+    """
+
+    # Setup a mocked Textlab API server with dummy responses
+    mock_server.configure("/user/login", 200, {"token": "dummy"})
+    mock_server.configure("/benchmark/420", 200, {"message": "Error occurred"})
+
+    # Redirect call aimed at the Textlab API to the fake server
+    settings.TEXTLAB_API_URL = f"http://localhost:{mock_server.port}"
+
+    # Enable Textlab in the test region
+    Region.objects.filter(slug="augsburg").update(hix_enabled=True)
+
+    edit_page, response = update_page_content(
+        admin_client, "Willkommen in Augsburg", "Neuer Inhalt2", True
+    )
```

Comment:
Slight preference from me: by the time I read to here I already forgot the fourth argument to `update_page_content()` and was confused about where you actually set the page to ignore the HIX score. If we pass it explicitly as a kwarg, this should be abundantly clear and no future Peter should be able to just overlook it. :grin: 

```suggestion
    edit_page, response = update_page_content(
        admin_client, "Willkommen in Augsburg", "Neuer Inhalt2", hix_ignore=True
    )
```